### ISSUE TITLE
Indent that work better with ternary operator

### ISSUE THREAD

**Issue by jefer94** [eslint/eslint#11537]
**What rule do you want to change?**
Indent

**Does this change cause the rule to produce more or fewer warnings?**
```bash
   26:1   error  Expected indentation of 10 spaces but found 8   indent
   27:1   error  Expected indentation of 12 spaces but found 10  indent
   28:1   error  Expected indentation of 10 spaces but found 8   indent
   29:1   error  Expected indentation of 12 spaces but found 8   indent
   30:1   error  Expected indentation of 14 spaces but found 10  indent
   31:1   error  Expected indentation of 12 spaces but found 8   indent
   32:1   error  Expected indentation of 14 spaces but found 8   indent
   33:1   error  Expected indentation of 16 spaces but found 10  indent
   34:1   error  Expected indentation of 16 spaces but found 10  indent
```
**How will the change be implemented? (New option, new default behavior, etc.)?**

New option in Indent rule

**Please provide some example code that this change will affect:**

```js
      const v = (
        id === 0 ?
          18 :
        id === 3 ?
          22 :
        id === 5 ?
          2 :
        id === 8 ?
          55 :
          null
      )
```

**What does the rule currently do for this code?**

Add 2 spaces by each ternary condition

**What will the rule do after it's changed?**

That don't add 2 spaces by each ternary condition

**Are you willing to submit a pull request to implement this change?**

Not
